Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1313,6 +1328,50 @@ void MediaPlayerPrivateGStreamer::commitLoad()
     updateStates();
 }
 
+bool MediaPlayerPrivateGStreamer::queryBufferingPercentage(GstBufferingMode& mode, int &percentage)
```

Comment:
No, I don't want. And I don't actually need any previous percentage value passed as a parameter. It was just a "tool" to use a variable defined in the caller scope (variable passed by reference) that I don't actually need, because the percentage is always overwritten.